torch/_inductor/codegen/triton.py

```diff
@@ -1310,12 +1310,19 @@ def index_expr(cls, expr, dtype):
         # we only respect non int32-int64 dtypes and otherwise use current kernel indexing dtype
         index_dtype = torch.int32 if V.kernel.index_dtype == "tl.int32" else torch.int64
         dtype = dtype if dtype not in (torch.int32, torch.int64) else index_dtype
-        var = V.kernel.cse.generate(
-            V.kernel.compute,
-            indexing.index_str,
-            bounds=get_bounds_index_expr(expr),
-            dtype=dtype,
-        )
+
+        # after we emit this var we cast it to the correct dtype
+        orig = config.test_configs.runtime_triton_dtype_assert
```

Comment:
Can use config.patch.

Reply:
the sub-fields dont contain that..

Reply:
Does the string form work for subfield?

like
config.patch("test_configs.runtime_triton_dtype_assert", False)